### ISSUE TITLE
Single hero image not showing up

### ISSUE THREAD

**Issue by thatryan** [WebDevStudios/wd_s#339]
Still trying to figure out why, when using content block template and attempting a hero block, with a single image, it will not appear in the viewport.

Block settings are,

![image](https://user-images.githubusercontent.com/26583/33337511-fd63cfc6-d427-11e7-95cc-478234cb58a2.png)

And on the front end the content appears pushed off screen to the right, 

![image](https://user-images.githubusercontent.com/26583/33337578-36d6312c-d428-11e7-8881-bd758800d1b6.png)

Oddly, I was checking slide output and put an `echo $slide_count;` in the block file here,

https://github.com/WebDevStudios/wd_s/blob/master/template-parts/content-blocks/block-hero.php#L11

and when doing that it (obviously) showed `1` but also brought the slide content into display properly...

**Comment by gregrickaby** [comment]
@thatryan Sorry you're having trouble.

When I try and re-create your issue, I'm unable to. My first though is try running `gulp` to make sure you have the latest Sass compiled in `style.css`.

![screenshot](https://dl.dropbox.com/s/fr34ldmd6wlguu5/Screenshot%202017-12-05%2008.59.14.png?dl=0)

![screenshot](https://dl.dropbox.com/s/y5no69yeymz9dvb/Screenshot%202017-12-05%2008.58.27.png?dl=0)

**Comment by thatryan** [comment]
So strange! 

Yeah I have run gulp several times :) 

Is the hero block the first/only thing on your test page? If I put a block above it it works perfectly..

The element, for me, has a 0 width. If I set width to 100% in dev-tools, it shows up. Not sure if that  gives you any ideas maybe..?

<img width="421" alt="cloudapp annotation 2017-12-05 at 10 04 48 am png" src="https://user-images.githubusercontent.com/26583/33622842-d245942e-d9a3-11e7-9c65-083282ab8a25.png">

**Comment by thatryan** [comment]
Ok I tracked down the issue! No idea WHY yet... but it was my fault. I moved the navigation out of the header to just below it so that it could be full width background. 

<img width="1199" alt="cloudapp annotation 2017-12-06 at 9 20 52 am png" src="https://user-images.githubusercontent.com/26583/33675247-c81ac4f6-da66-11e7-9439-d732adcc1704.png">

For some reason that breaks the hero. But only above 900px when it gets set to `display:block` 
Not sure why yet, but can close this as it is not a problem with the code, just my setup.
